OpenSim/Actuators/Test/testMuscleFixedWidthPennationModel.cpp

```diff
@@ -795,13 +796,30 @@ int main(int argc, char* argv[])
         cout << "**************************************************" << endl;
         cout << "TEST: Exception Handling" << endl;
 
-        //constructor
-        SimTK_TEST_MUST_THROW(MuscleFixedWidthPennationModel fibKinEX = 
-            MuscleFixedWidthPennationModel(0, optPenAng, SimTK::Pi/2.0));
-        SimTK_TEST_MUST_THROW(MuscleFixedWidthPennationModel fibKinEX = 
-            MuscleFixedWidthPennationModel(optFibLen, -0.01, SimTK::Pi/2.0));
-        SimTK_TEST_MUST_THROW(MuscleFixedWidthPennationModel fibKinEX = 
-            MuscleFixedWidthPennationModel(optFibLen, SimTK::Pi/2, SimTK::Pi/2.0));
+        // Test property bounds.
+        {
+            // TODO: Uncomment and refine when MuscleFixedWidthPennationModel is
+            //       a property of Thelen2003Muscle.
```

Comment:
Is this comment accurate? Should the pennation model be a property of the Muscle?